README.md

```diff
@@ -9,6 +9,8 @@ The combination of the image manifest, image configuration, and one or more file
 
 The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
 
+The keywords "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][unspecified].
```

Comment:
On Wed, Sep 14, 2016 at 02:58:04AM -0700, Jonathan Boulle wrote:
> > +The keywords "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][unspecified].
>
> "unspecified" is an awkward name for a link (especially a reference
> one because at first blush it looks like a placeholder) - can you
> call this c99rationale or similar?

In case we ever link to another part of the c99 rationale, I've gone
with c99-unspecified in 01687e9 → 2d49333.